src/NoTimeToWaitBundle/Resources/views/Default/nttw3.html.twig

```diff
@@ -1,40 +1,58 @@
 {% extends 'NoTimeToWaitBundle::base.html.twig' %}
 
-{% block title %}No Time to Wait 3: {% endblock %}
+{% block title %}No Time to Wait 3{% endblock %}
 
 {% block body %}
-<h1><strong>No Time to Wait!</strong> *</h1>
+<h1>* <strong>No Time to Wait!</strong> *</h1>
```

Comment:
Asterisks for ...decoration?

Reply:
the asterisk on the right was taken from nttw1 & 2, the meaning is at the bottom. No change compared to last year ;-).
So the question is more about why the asterisk on the left, of if we should remove the asterisk on the right.